test/testNotificationsMultipleMachines.js

```diff
@@ -22,30 +24,39 @@ describe('mercurius (multiple-machines-)notify', function() {
     });
   });
 
+  afterEach(function(done) {
+    nock.cleanAll();
```

Comment:
It'd be better to do something like:
```
assert.true(nock.isDone(), 'test finished without pending requests: ' + nock.pendingMocks());
```
So you don't need to check in each individual test.